cmd/core/sdf_filter_parser.go

```diff
@@ -0,0 +1,124 @@
+package core
+
+import (
+	"fmt"
+	"net"
+	"regexp"
+	"strconv"
+
+	"github.com/edgecomllc/eupf/cmd/ebpf"
+)
+
+func ParseSdfFilter(flowDescription string) (ebpf.SdfFilter, error) {
```

Comment:
We have `SdfFilter` structure here from ebpf package. The point is this code should be isolated from ebpf details. 

In core package we can just store sdf filter as suitable in golang. Later during write to ebpf map we can translate suitable go struct to specific ebpf struct.

Reply:
Does this mean that in ebpf/pdr.go will be imported "github.com/edgecomllc/eupf/cmd/core"?

Reply:
Oh, and should PdrInfo also be moved from ebpf/ to core/? Because it's also violation the rules of core-ebpf isolation. For instance, "github.com/edgecomllc/eupf/cmd/ebpf" is imported in core/session.go

Reply:
Exactly

Reply:
Should I move SdfFilter, PdrInfo, FarInfo, QerInfo to session.go? Or should I create a file core/session_info.go just to store structs?

Reply:
Let's postpone this point